relay-event-derive/src/lib.rs

```diff
@@ -90,7 +91,12 @@ fn derive_process_value(mut s: synstructure::Structure<'_>) -> TokenStream {
 
         let mut body = TokenStream::new();
         for (index, bi) in variant.bindings().iter().enumerate() {
-            let field_attrs = parse_field_attributes(index, bi.ast(), &mut is_tuple_struct);
+            let mut field_attrs = parse_field_attributes(index, bi.ast(), &mut is_tuple_struct);
+            // In case the type has the attribute `trim` set to false, we want to inject this
+            // information in the field attributes so that it can be read by the processor.
+            if let TrimmingMode::Disabled = type_attrs.trim {
+                field_attrs.trim = Some(type_attrs.trim);
+            }
```

Comment:
Could this be an unconditional

```suggestion
            field_attrs.trim = matches!(type_attrs.trim, TrimmingMode::Enabled);
```

Reply:
Yes!